manim/utils/file_ops.py

```diff
@@ -56,4 +58,19 @@ def get_sorted_integer_files(directory,
         elif remove_non_integer_files:
             os.remove(full_path)
     indexed_files.sort(key=lambda p: p[0])
-    return list(map(lambda p: os.path.join(directory, p[1]), indexed_files))
\ No newline at end of file
+    return list(map(lambda p: os.path.join(directory, p[1]), indexed_files))
+
+def open_file(file_path):
+    current_os = platform.system()
+    if current_os == "Windows":
+        os.startfile(file_path)
+    else:
+        commands = []
+        if current_os == "Linux":
+            commands.append("xdg-open")
+        elif current_os.startswith("CYGWIN"):
+            commands.append("cygstart")
+        else:  # Assume macOS
```

Comment:
**Nitpick (if-minor)** : For macOs (but you surely do know that already), name of the os is `Darwin`.
I think it would be better to change it into and if, and to raise an error or to do something by default if the os is not found (in theory it should never append, as platform.system() return `''` if the system is not found and only returns four values.

Reply:
Thank you for the suggestion! I've made it raise an `OSError` if the OS is not recognisable.